docs/trendz/install/resources/1.10.3/trendz-centos-download.sh

```diff
@@ -0,0 +1 @@
+wget https://dist.thingsboard.io/trendz-1.10.3.rpm
```

Comment:
Please refactor this to use configuration variable. this will simplify releases. See ```_config.yml``` and docs/user-guide/install/resources/thingsboard-centos-installation.sh